extension/tests/test-fixtures.ts

```diff
@@ -0,0 +1,35 @@
+import { test as base, chromium, BrowserContext } from "@playwright/test";
+import path from "path";
+
+export const test = base.extend<{
+  context: BrowserContext;
+  extensionId: string;
+}>({
+  context: async ({}, use) => {
+    const pathToExtension = path.join(__dirname, "../build");
+    console.log(pathToExtension);
```

Comment:
console.log

Reply:
removed in e99194053f8fb93000911ce5cb43589f3ef43894